controllers/discovery/initDiscoveryUserData.js

```diff
@@ -2,7 +2,7 @@ const {sequelize, UserBlockedUser} = require('../../databases/models');
 const _ = require('lodash');
 const {QueryTypes} = require('sequelize');
 const UsersFunction = require('../../databases/functions/users');
-const {MINIMUM_KARMA_SCORE} = require('../../helpers/constants');
+const {MINIMUM_KARMA_SCORE, MAX_EXCLUDED_USERS} = require('../../helpers/constants');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `MAX_EXCLUDED_USERS` has been added correctly.
```diff
- const {MINIMUM_KARMA_SCORE} = require('../../helpers/constants');
+ const {MINIMUM_KARMA_SCORE, MAX_EXCLUDED_USERS} = require('../../helpers/constants');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->